### PR TITLE
DevEx: Filter userid

### PR DESCRIPTION
**Pull request by ericsorenson**

### CODE REVIEW

**Review by rachaelparris — APPROVED** [review]

File: shared/src/business/entities/cases/Case.js [hunk 1]
```diff
@@ -284,6 +283,9 @@ function Case(rawCase, { applicationContext, filtered = false }) {
   this.trialSessionId = rawCase.trialSessionId;
   this.trialTime = rawCase.trialTime;
 
+  if (applicationContext.getCurrentUser().userId === rawCase.userId)
+    this.userId = rawCase.userId;
```
Comment (matthopson) [inline shared/src/business/entities/cases/Case.js:287]: Did you intend to leave the brackets off? I know it's valid, but not common in our codebase.

**Review by matthopson — APPROVED** [review]
One small code style question, otherwise looks good 👍

File: shared/src/business/entities/Document.js [hunk 2]
```diff
@@ -417,16 +448,22 @@ joiValidationDecorator(
...
     receivedAt: joi.date().iso().optional(),
-    relationship: joi.string().optional(),
-    scenario: joi.string().optional(),
+    relationship: joi
+      .string()
+      .valid(...Object.values(Document.RELATIONSHIPS))
+      .optional(),
+    scenario: joi
+      .string()
+      .valid(...Object.values(Document.SCENARIOS))
```
Comment (kkoskelin) [inline shared/src/business/entities/Document.js]: Both of these things are already arrays, not objects -- perhaps this can be simplified?

**Review by kkoskelin — APPROVED** [review]
Just one observation where we are unnecessarily using `Object.values` in a couple places, but no show-stoppers.

File: shared/src/business/entities/Document.js (hunk 2 above)
Comment (ericsorenson) [reply shared/src/business/entities/Document.js]: I'm open to suggestions!

File: shared/src/business/entities/cases/Case.js (hunk 1 above)
Comment (ericsorenson) [reply shared/src/business/entities/cases/Case.js:287]: Fixed!

File: shared/src/business/entities/Document.js (hunk 2 above)
Comment (kkoskelin) [reply shared/src/business/entities/Document.js]: I think you can just splat-in the values, i.e.
`.valid(...Document.RELATIONSHIPS)`

**Review by kkoskelin — APPROVED** [review]

**Pull request merged**